boards/arm/nrf52dk_nrf52805/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# The nrf52dk_nrf52805 board mirrors the nRF52832 DK hardware. This
```

Comment:
Strictly speaking, there is no "nRF52832 DK" hardware. I know we have the same line in nrf52810, but being pedantic, this should be nRF 52 DK

Reply:
Fixed.